database/API.py

```diff
@@ -43,4 +43,18 @@ def features_get_by_sno():
     for feature in session.scalars(stmt):
         result.append(feature.as_dict())
     
-    return result, 200
\ No newline at end of file
+    return result, 200
+
+@app.route('/features/create-region', methods=['POST'])
```

Comment:
Suggestion (non-blocking): some of our functions have a docstring and this one does not. It is not necessary but I think docstrings can be very useful for a dev who hasn't worked on the backend yet